src/KafkaFlow.Abstractions/EmptySerializationContext.cs

```diff
@@ -0,0 +1,15 @@
+namespace KafkaFlow
```

Comment:
What do you think about we create a `SerializationContext` class with an `Empty` property?

```csharp
public class SerializerContext : ISerializerContext
  {
    public static SerializerContext Empty => new SerializerContext();
  }
```

So, we would work always with SerializationContext (no need of having an 'EmptySerializationContext') and SerializationContext.Empty would be the default context for now

Reply:
Agreed!